### DISCUSSION TITLE
Intro to Numpy

### DISCUSSION THREAD

**Discussion by giscus** [YLu-1258/YLU_blog#2] [Comments]
# Intro to Numpy

NumPy

https://ylu-1258.github.io/YLU_blog/posts/Numpy-for-Beginners/

**Comment by YLu-1258** [comment]
Testing Testing hey hey

**Reply by YLu-1258** [reply]
Testy reply hey hey

**Reply by YLu-1258** [reply]
Even more replies hey hey
